src/cnxml/document.js

```diff
@@ -156,17 +156,67 @@ export const LIST_ITEM = block('item', de_list_item, 'list_item', 'item')
 /**
  * Process data for media nodes.
  */
-function media(el, next) {
+function de_media(el, next) {
+    const alt = el.getAttribute('alt')
+
+    const nodes = next(Array.from(el.children))
+
+    if (alt) {
+        nodes.push({
+            object: 'block',
+            type: 'media_alt',
+            nodes: [{
+                object: 'text',
+                text: alt,
+            }],
+        })
+    }
+
     return {
         type: 'media',
-        data: {
-            alt: el.getAttribute('alt'),
-        },
-        nodes: next(Array.from(el.children)),
+        nodes: nodes,
+    }
+}
+
+/**
+ * Deserializer for divs.
+ */
+function de_div(el, next) {
+    const type = el.getAttribute('class') || 'div'
+
+    return mixedContent(el, next, type.replace('-', '_'))
```

Comment:
We should only support loading of `div`s with known class and properties, otherwise you can use them to insert any content into the model.